Lombiq.ContentEditors.Tests.UI/Lombiq.ContentEditors.Tests.UI.csproj

```diff
@@ -21,7 +21,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(NuGetBuild)' == 'true'">
-    <PackageReference Include="Lombiq.Tests.UI" Version="6.0.2-alpha.2.osoe-548" />
+    <PackageReference Include="Lombiq.Tests.UI" Version="6.0.2-alpha.1.osoe-641" />
```

Comment:
It doesn't matter anymore, but this should've rather been `6.0.2-alpha.3.osoe-641`. Now it'll be harder to update. See: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/786857987/Managing+releases+of+open-source+projects#:~:text=When%20publishing%20a%20pre%2Drelease%20version%20for%20something%20temporary